api.lua

```diff
@@ -95,36 +95,55 @@ function areas:getAreasIntersectingArea(pos1, pos2)
 	return res
 end
 
--- Checks if the area is unprotected or owned by you
+-- Returns smallest area at position and its id or nil.
+-- If multiple areas have the same volume, larger id takes precedence.
+function areas:getSmallestAreaAtPos(pos)
+	local smallest_area, smallest_id, smallest_volume, volume = nil
```

Comment:
1. Misleading `= nil`. That only affects the first variable.
2. `local volume` might not bring the performance advantage that you're hoping for. Lua can simply put the number in L103 onto its stack and work with that. AFAIK there's no allocation optimization needed in such cases. Keeping it as-is should not hurt performance either.

Suggestion is a combination of the subsequent comments:
```suggestion
	local smallest_area, volume
	local smallest_volume = math.huge
```

Reply:
Thanks, does look neater. Addressed in https://github.com/minetest-mods/areas/pull/79/commits/56763e16f9b262059782935c27adb91bd84a3c99